### PR TITLE
Extra capabilities

### PR DESCRIPTION
**Pull request by caroldelwing**
## Describe the Change

This PR adds a "Palette Additional Capabilities" page to the new Getting Started section as outlined in Approach 2 of the Revise and Link Existing Content RFC.

## Review Changes
💻 [Getting Started](https://deploy-preview-2354--docs-spectrocloud.netlify.app/getting-started/)
💻 [Palette Additional Capabilities](https://deploy-preview-2354--docs-spectrocloud.netlify.app/getting-started/additional-capabilities)

🎫 [DOC-1066](https://spectrocloud.atlassian.net/browse/DOC-1066)

[DOC-1066]: https://spectrocloud.atlassian.net/browse/DOC-1066?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Palette offers a range of additional capabilities designed to extend the functionality and versatility of Kubernetes
+workloads. These features comprise diverse deployment scenarios, ensuring organizations can manage their workloads
+across different environments. This can include data centers such as VMware vSphere and OpenStack, bare metal, or public
+cloud using your cloud accounts.
+
+Whether deploying workloads in remote locations with [Edge](./additional-capabilities.md#edge), securing sensitive
+information with [VerteX](./additional-capabilities.md#palette-vertex), maximizing management control with
+[self-hosted Palette](./additional-capabilities.md#self-hosted-palette), or integrating virtual machine workloads into
+Kubernetes environments with [Virtual Machine Orchestrator](./additional-capabilities.md#virtual-machine-orchestrator),
+Palette provides comprehensive solutions to meet different business needs. Learn more about each capability below.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:20]:
```suggestion
Whether deploying workloads in remote locations with [Edge](./additional-capabilities.md#edge), enforcing FIPS compliance with [VerteX](./additional-capabilities.md#palette-vertex), maximizing management control with
[self-hosted Palette](./additional-capabilities.md#self-hosted-palette), or integrating virtual machine workloads into
Kubernetes environments with [Virtual Machine Orchestrator](./additional-capabilities.md#virtual-machine-orchestrator),
Palette provides comprehensive solutions to meet different business needs. Learn more about each capability below.
```
I focused on FIPS compliance versus securing sensitive information, which could get us into the data encryption rabit hole. We can avoid that by focusing on the FIPS aspect 😄

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+![A drawing of Palette with humans interacting](/getting-started/getting-started_additional-capabilities_palette.png)
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:22]: Awesome image, but can we make it a bit smaller? It's really big 😅

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Edge
+
+Palette Edge enables you to deploy Kubernetes workloads in remote or rural locations without having to provision
+high-bandwidth connectivity from these sites back to the cloud. Its secure and decentralized architecture allows
+services to operate even without connection to the central management plane while enforcing policies locally.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:28]: The second sentence feels a bit repetitive, maybe we can focus on other Edge's abilities? Such support for different workloads, flexible install, amd64/arm64 architectures support,  or multiple k8s distro support? Edge deployments are also immutable thanks to Kairos -that's another important feature.

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Edge clusters are Kubernetes clusters set up on Edge hosts. These hosts can be bare metal or virtual machines located in
+isolated locations. Palette deploys and manages workload clusters at the Edge through the Palette management console,
+bringing computing and data storage closer to the source. This approach reduces latency and bandwidth issues that result
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:32]:
```suggestion
isolated locations. Palette deploys and manages workload clusters at the Edge. You can manage Edge clusters locally on-site through the Edge Management console, or centrally through the Palette management plane. 
Palette Edge is able to meet your needs, regardless of the network topology your deployments face. 
```

We gotta be a bit careful here. Edge can be managed locally through the EMC (4.3) feature if in an airgap environment. If not, then you can use the default Palette UI.

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+from central computing, enhancing overall application performance. Learn more on the
+[Palette Edge](../clusters/edge/edge.md) page.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:34]:
```suggestion
Check out the
[Palette Edge](../clusters/edge/edge.md) page to learn more about Edge and its features.
```

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+## Self-Hosted Palette
+
+Palette is available as a fully self-hosted platform offering. You can install an instance of the Palette management
+plane in your data center or public cloud provider to manage Kubernetes clusters.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:39]:
```suggestion
Palette is available as a fully self-hosted platform offering. You can install an instance of the Palette management
plane in your data center or public cloud provider and use it to manage all your Kubernetes clusters.
```

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+This option gives you maximum control over the platform, as you are responsible for the day-to-day management.
+Additionally, it enables deployments in air-gapped environments where access to the public cloud is not possible. Read
+more on the [Self-Hosted Palette](../enterprise-version/enterprise-version.md) page.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:43]: Hmm...focusing on the day-to-day management is not the most attractive feature 😅  Below are some ideas on what we can focus on:

- Meet compliance and/or legal requirements. Hosting Palette ensure they can answer all questions about how their workloads are managed, who has access, and where internal system data is stored.
- Upgrade at their own pace
- Tenant management
- Infrastructure management and ability to choose different hosting models (vSphere, k8s)

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+## Palette VerteX
+
+Palette VerteX offers a simple, flexible, and secure way for government and regulated industries to manage Kubernetes
+workloads containing sensitive and classified information. It is available as a self-hosted platform offering that you
+can install in your data center or public cloud provider.
+
+Palette VerteX incorporates validated Federal Information Processing Standards (FIPS) 140-2 cryptographic modules into
+the Kubernetes clusters it deploys, ensuring robust data protection for your organization’s infrastructure and
+applications. Read more on [Palette VerteX](../vertex/vertex.md) page.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:53]:
```suggestion
applications. To learn more, check out the [Palette VerteX](../vertex/vertex.md) documentation.
```
A bit smoother transition

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+## Virtual Machine Orchestrator
+
+Palette Virtual Machine Orchestrator (VMO) allows you to deploy and manage traditional VM workloads within a modern
+Kubernetes environment, side by side with your containerized applications. With VMO, you can deploy, manage, and scale
+VMs within the same cluster that hosts your containerized applications. VMs are managed as Kubernetes pods, ensuring
+complete mapping between the VM and Kubernetes concepts.
+
+Palette VMO simplifies infrastructure management and eliminates the need for a hypervisor by leveraging Canonical’s
+Metal As A Service (MAAS). It works in self-hosted, air-gapped, and Spectro Cloud’s SaaS environments. Learn more on the
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:63]:
```suggestion
Metal As A Service (MAAS). It works in self-hosted, air-gapped, and in our SaaS environments. Learn more on the
```

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Hey @caroldelwing really good start but we have some opprtuntieis for improvement. Take a peek at the feedback below. Additionaly, touch base with the following PMs for each respective section:

- Edge - Anton
- VerteX - Tom
- VMO - Romain

I can work with you on self-hosted.

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 1 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:20]: Oooo. I see :eyes:

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 2 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:21]: Yep, done :)

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 3 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:27]: Added other Edge's abilities :white_check_mark:

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 7 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:42]: Agree :) Nice ideas! I incorporated the features.

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+sidebar_label: "Palette Additional Capabilities"
+title: "Palette Additional Capabilities"
+description: "Learn more about Palette's Additional Capabilities."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["getting-started"]
+---
+
+Palette offers a range of additional capabilities designed to extend the functionality and versatility of Kubernetes
```
Comment (antongisli) [inline docs/docs-content/getting-started/additional-capabilities.md:11]: are we really extending the functionality and versatilty of the workloads themselves? 

or are we extended the ability for palette users to deploy and manage workloads more effectively across a broad range of cloud types (etc)

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+workloads. These features comprise diverse deployment scenarios, ensuring organizations can manage their workloads
+across different environments. This can include data centers such as VMware vSphere and OpenStack, bare metal, or public
```
Comment (antongisli) [inline docs/docs-content/getting-started/additional-capabilities.md:13]: please mention Canonical MAAS for bare metal specifically, perhaps worth linking also?

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+cloud using your cloud accounts.
+
+Whether deploying workloads in remote locations with [Edge](./additional-capabilities.md#edge), enforcing FIPS
```
Comment (antongisli) [inline docs/docs-content/getting-started/additional-capabilities.md:16]: With edge, the mantra is:
many smaller clusters - i.e. the challenge is in the volume (count) of clusters, vs. the size of the clusters. Might want to add managing Edge locations at scale with potentially 1000s of sites.

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+compliance with [VerteX](./additional-capabilities.md#palette-vertex), maximizing management control with
+[self-hosted Palette](./additional-capabilities.md#self-hosted-palette), or integrating virtual machine workloads into
+Kubernetes environments with [Virtual Machine Orchestrator](./additional-capabilities.md#virtual-machine-orchestrator),
+Palette provides comprehensive solutions to meet different business needs. Learn more about each capability below.
+
+![A drawing of Palette with humans interacting](/getting-started/getting-started_additional-capabilities_palette.png)
+
+## Edge
+
+Palette Edge enables you to deploy Kubernetes workloads in remote or rural locations without having to provision
```
Comment (antongisli) [inline docs/docs-content/getting-started/additional-capabilities.md:26]: the term "a diverse range of locations, including industrial, rural, finance, retail, etc..."
Remote or rural doesn't capture the broad range of places that edge lcusters will be deployed to

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+high-bandwidth connectivity from these sites back to the cloud. It supports both VM and container-based workloads,
+multiple Kubernetes distributions, and Intel and ARM hardware architectures. Palette Edge is built on top of the
+open-source project [Kairos](https://kairos.io/), which provides an immutable and highly available operating system with
```
Comment (antongisli) [inline docs/docs-content/getting-started/additional-capabilities.md:29]: Kairos doesn't provide an OS - we should say that it enables the creation of immutable versions of operating systems such as Ubuntu as well as easy customization of the OS.

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+zero-downtime rolling upgrades. Additionally, it is designed to scale to tens of thousands of locations while enforcing
+policies locally within each cluster.
+
+Edge clusters are Kubernetes clusters set up on Edge hosts. These hosts can be bare metal or virtual machines located in
+isolated locations. Palette deploys and manages workload clusters at the Edge, and the services continue operating even
+when the connection to the management plane is lost. You can manage Edge clusters locally on-site through the Edge
+Management console, or centrally through the Palette management plane. Palette Edge is able to meet your needs,
```
Comment (antongisli) [inline docs/docs-content/getting-started/additional-capabilities.md:36]: please use the term local UI (not EMC)

**Review by antongisli — COMMENTED** [review]
finished my review. I think this document might benefit from a quick reveiw from Mr Ant Newman also.

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 11 above)
Comment (cloudmaniac) [inline docs/docs-content/getting-started/additional-capabilities.md:13]: Do we want to mention OpenStack at all? Thoughts @antongisli ?

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+
+## Palette VerteX
+
+Palette VerteX offers a simple, flexible, and secure way for government and regulated industries to manage Kubernetes
+workloads containing sensitive and classified information. It is available as a self-hosted platform offering that you
+can install in your data center or public cloud provider.
+
+Palette VerteX incorporates validated Federal Information Processing Standards (FIPS) 140-2 cryptographic modules into
+the Kubernetes clusters it deploys, ensuring robust data protection for your organization’s infrastructure and
+applications. To learn more, check out the [Palette VerteX](../vertex/vertex.md) documentation.
```
Comment (cloudmaniac) [inline docs/docs-content/getting-started/additional-capabilities.md:61]: The other major benefit of VerteX is that the management plane is FIPS compliant, not only the target clusters it deploys. We should start with this IMHO.

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 10 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:11]: Excellent point. I rewrote the paragraph.

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 11 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:13]: :white_check_mark:

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 12 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:16]: Nice! I rewrote the sentence.

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 15 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:36]: :white_check_mark:

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 11 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:13]: I removed OpenStack :)

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 14 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:29]: Fixed it :)

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 16 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md:61]: :white_check_mark:

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+## Palette VerteX
+
+Palette VerteX offers a simple, flexible, and secure way for government and regulated industries to manage Kubernetes
+workloads containing sensitive and classified information. It is available as a self-hosted platform offering that you
+can install in your data center or public cloud provider.
+
+Palette VerteX incorporates validated Federal Information Processing Standards (FIPS) 140-2 cryptographic modules into
+its management plane and the Kubernetes clusters it deploys, ensuring robust data protection for your organization’s
+infrastructure and applications. To learn more, check out the [Palette VerteX](../vertex/vertex.md) documentation.
+
```
Comment (chainsaw2k) [inline docs/docs-content/getting-started/additional-capabilities.md:65]: Integrating some of my other text with yours, I came up with something like this:

- Palette VerteX offers a simple, flexible, and secure way for government and regulated industries to manage Kubernetes workloads containing sensitive and classified information. It is available as a self-hosted platform offering that you can install in your secure data center or public cloud provider.
- Palette is fully proven in operational environments giving it a TRL 9 designation, suitable for use with workloads in production on the high-security side up to IL 5/6/6+ environments. It enables you to deploy and manage the full life-cycle of multiple Kubernetes clusters in government clouds such as AWS and Azure, and in air-gap data centers such as vSphere and Nutanix. VerteX also allows you to operate edge nodes where intermittent connectivity, low bandwidth, or complete disconnection pose a challenge.
- Palette VerteX incorporates validated Federal Information Processing Standards (FIPS) 140-2 cryptographic modules into its management plane and the Kubernetes clusters it deploys, ensuring robust data protection for your organization’s infrastructure and applications. TLS 1.3 is used throughout.
- A suite of scanning tools and CONUS support from a dedicated public sector team round out the offering. To learn more, check out the Palette VerteX documentation.

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 18]
```diff
@@ -23,34 +23,56 @@ Palette provides comprehensive solutions to meet different business needs. Learn
...
-workloads containing sensitive and classified information. It is available as a self-hosted platform offering that you
-can install in your data center or public cloud provider.
+Palette VerteX offers a simple, flexible, and secure way for government and regulated industries to deploy and manage
+Kubernetes workloads containing sensitive and classified information. It is available as a self-hosted platform offering
+that you can install in various environments. These include virtualized and bare metal data centers (like
+[VMware vSphere](https://www.vmware.com/products/vsphere.html) and [Nutanix](https://www.nutanix.com/)), clouds
+(including [AWS](https://aws.amazon.com/govcloud-us/) and
+[Azure](https://azure.microsoft.com/en-ca/explore/global-infrastructure/government) government clouds), and edge
+locations (including air-gapped setups), which makes VerteX also appropriate for addressing challenges like intermittent
+connectivity or low bandwidth.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:65]: @caroldelwing I feel like we are mixing two concepts here. The hosting model of VerteX, and what clusters you can deploy through VerteX.

File: docs/docs-content/getting-started/additional-capabilities.md [hunk 19]
```diff
@@ -59,6 +81,6 @@ Kubernetes environment, side by side with your containerized applications. With
 VMs within the same cluster that hosts your containerized applications. VMs are managed as Kubernetes pods, ensuring
 complete mapping between the VM and Kubernetes concepts.
 
-Palette VMO simplifies infrastructure management and eliminates the need for a hypervisor by leveraging Canonical’s
-Metal As A Service (MAAS). It works in self-hosted, air-gapped, and Spectro Cloud’s SaaS environments. Learn more on the
+Palette VMO simplifies infrastructure management and eliminates the need for a hypervisor by leveraging
+[Canonical MAAS](https://maas.io). It works in self-hosted, air-gapped, and in our SaaS environments. Learn more on the
```
Comment (karl-cardenas-coding) [inline docs/docs-content/getting-started/additional-capabilities.md:85]:
```suggestion
[Canonical MAAS](https://maas.io). It works in self-hosted, airgapped, and in our SaaS environments. Learn more on the
```

**Review by karl-cardenas-coding — COMMENTED** [review]
Looks great, I just have one concern in the VerteX section.

File: docs/docs-content/getting-started/additional-capabilities.md (hunk 18 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/additional-capabilities.md]: Yes, you're right :sweat_smile: I've separated the concepts and placed them in different paragraphs.

**Pull request merged**